test/robot/build/package.go

```diff
@@ -173,11 +182,25 @@ func (pkg *Package) mergeTool(ctx context.Context, tool *ToolSet) {
 }
 
 // GetTools will return the toolset that match the abi, if there is one.
```

Comment:
`// GetHostTools will return the host toolset that matches the abi, if there is one.`

Would have been caught with VSCode and the go plugin.